presto-docs/src/main/sphinx/release/release-0.204.rst

```diff
@@ -0,0 +1,43 @@
+=============
+Release 0.203
+=============
+
+General
+-------
+* Fix a memory tracking bug that causes the memory to be overestimated for scan operators when reading ORC files.
+* Use distributed join if one side is naturally partitioned on join keys.
+* Improve performance of correlated subqueries when filters from outer query can be propagated to the subquery.
+* Improve performance for correlated subqueries that contain inequalities.
+* Add support for all geometry types in :func:`ST_Area`.
+* Add :func:`ST_EnvelopeAsPts` function.
+* Add :func:`to_big_endian_32` and :func:`from_big_endian_32` functions.
+* Add cast between ``VARBINARY`` type and ``IPADDRESS`` type.
+* Make :func:`lpad` and :func:`rpad` functions support varbinary in addition to varchar.
+* Fill in with ``NULL`` when processing arrays of different length in :func:`zip_with`.
```

Comment:
How about
```
* Allow using arrays of mismatched lengths with :func:`zip_with`.
  The missing positions are filled with ``NULL``.
```
Also, fix the documentation for `zip_with` in `array.rst`. The "NULL" part only has single backticks but needs to have double.